dfdatetime/time_elements.py

```diff
@@ -907,7 +907,7 @@ def __init__(
 
     Args:
       fraction_of_second (Optional[decimal.Decimal]): fraction of second, which
-          must be a value between 0.0 and 1.0.
+          must be a value between 0.00 and 1.00.
```

Comment:
why this change?

Reply:
I first changed it in the doc string to make it more obvious to the reader that more digits were permitted. It is pointless to have it in the code. I'll revert it.

Reply:
Noworries, I'll make the changes.